hawk/app/views/shared/_simulator.html.haml

```diff
@@ -1,11 +1,11 @@
 #simulator-node
 
 %script#simulator{type: "text/x-jsrender"}
-  .panel.panel-default
+  .panel.panel-warning
     #simulator-heading.panel-heading
       %h3.panel-title
-        %a{href: "#simulator-collapse", role: :button, data: {toggle: :collapse, parent: "#simulator"}, aria: {expanded: true, controls: "simulator-collapse"}}
-          = _("Simulator")
+        %a#simulator-toggle{href: "#simulator-collapse", role: :button, data: {toggle: :collapse, parent: "#simulator"}, aria: {expanded: true, controls: "simulator-collapse"}}
```

Comment:
Line is too long. [176/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace